lib/Form/Checkbox/styles.scss

```diff
@@ -0,0 +1,17 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+$form-checkbox-margin: $layout-spacing-base/2 !default;
+$form-checkbox-font-weight: 400;
```

Comment:
@kyleharper There's not supposed to be... I've checked and there's a master merge that somehow brought in specific `filecard-component` into it, not sure what happened with that. 

But the merge is clean and should be good to go regardless of that (I'll investigate this further though, quite curious!). Only those 3 commits regarding these small components. I'll add the variable for the type weight, completely missed that one 👍